tests/git.rs

```diff
@@ -971,7 +971,7 @@ fn stale_cached_version() {
     let rev = repo.revparse_single("HEAD").unwrap().id();
 
     File::create(&foo.root().join("Cargo.lock")).unwrap().write_all(format!(r#"
-        [root]
+        [[package]]
```

Comment:
Is it true that changes in tests are mostly cosmetics? That is, the tests pass with `[root]` as well as with `[[package]]`?

Reply:
Looks like it. I made the same change against master branch and the test passes without a hitch